src/invariant/LedgerEntryIsValid.cpp

```diff
@@ -107,7 +107,11 @@ LedgerEntryIsValid::checkIsValid(LedgerEntry const& le,
     case DATA:
         return checkIsValid(le.data.data(), version);
     case CLAIMABLE_BALANCE:
-        return checkIsValid(le, previous, version);
+        if (le.ext.v() != 1 || !le.ext.v1().sponsoringID)
+        {
+            return "ClaimableBalance is not sponsored";
+        }
+        return checkIsValid(le.data.claimableBalance(), previous, version);
     case LIQUIDITY_POOL:
         return checkIsValid(le.data.liquidityPool(), version);
```

Comment:
Shouldn't we be checking that the liquidity pool has `le.ext.v() == 0`?

Reply:
Yeah good point. Fixed.